HLeithner/joomla-cms
# Add prepared statements for plg-installer-override

### Summary of Changes
Updated SQL queries to prepared statements and made some cleanups around the queries.


### Testing Instructions
Test instruction could be found in the original PR:
https://github.com/joomla/joomla-cms/pull/21851

> Video: https://www.youtube.com/watch?v=l0_jFjswObI
For this, you need an extension which has an update with new changes in extension files.
So,
You can use these components
com_helloworld.0.0.1 https://www.dropbox.com/s/106cgn1lclp9dxm/com_helloworld.0.0.1.zip?dl=0
com_helloworld.0.0.2 https://www.dropbox.com/s/xhl8p0nz3wdvapj/com_helloworld.0.0.2.zip?dl=0
Download this branch/PR then install fresh Joomla using this branch/PR.
Go to the template manager.
Create an override of that extension before the update.
See the diff view of that file by using a toggle button in the top right corner. ** Expected:** No difference.
Now update your extension. ** Expected:** If your created override file got any update in core you can see notification message number of files updated.
Go to template manager check new tab you can see which files are updated.
Click on the name of the file you will get redirect to edit view of that file.
Now you can see the changes in diff view.
Adjust your override file if you want and then you can delete that file list history or mark as checked.


### Expected result
Nothing changed.
